yarn.lock

```diff
@@ -4,28 +4,28 @@
 
 "@ampproject/remapping@^2.2.0":
   version "2.3.0"
-  resolved "https://registry.yarnpkg.com/@ampproject/remapping/-/remapping-2.3.0.tgz#ed441b6fa600072520ce18b43d2c8cc8caecc7f4"
+  resolved "https://registry.npmjs.org/@ampproject/remapping/-/remapping-2.3.0.tgz"
```

Comment:
did you run npm install instead of yarn

Reply:
i swear i ran yarn install 😳 what should i do to fix it? delete and run yarn install?

Reply:
fixed